tools/winmd2objc/UnitTests/CodeGenTests.cpp

```diff
@@ -1,6 +1,24 @@
-#include "../lib/CodeGenModel.h"
+//******************************************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
 #include "stdafx.h"
 #include <gtest/gtest.h>
+#include <CodeGenModel.h>
+
+using namespace std;
```

Comment:
I agree, but this isn't worth the effort to do this as it's a mass usage across the cpp files.
It was also being used in the header files, I went ahead and removed that already.